lms/envs/common.py

```diff
@@ -392,6 +392,9 @@
     # Whether to check the "Notify users by email" checkbox in the batch enrollment form
     # in the instructor dashboard.
     'BATCH_ENROLLMENT_NOTIFY_USERS_DEFAULT': True,
+
+    # Whether the bulk enrollment view is enabled.
+    'ENABLE_BULK_ENROLLMENT_VIEW': False,
```

Comment:
Ah, right -- maybe all features are already being imported from the environment in one fell swoop